examples/enums-and-unions/resolver.ts

```diff
@@ -24,7 +24,7 @@ export class ExampleResolver {
   }
 
   @Query(_returns => [SearchResult])
-  async search(@Arg("cookName") cookName: string): Promise<Array<typeof SearchResult>> {
+  async search(@Arg("cookName") cookName: string): Promise<(typeof SearchResult)[]> {
```

Comment:
I don't like this syntax for complex types (with brackets). TSLint had a setting for simple and complex types. Can you check if we can set this here that way too?